src/main/java/net/hypixel/nerdbot/api/database/model/user/DiscordUser.java

```diff
@@ -64,10 +66,17 @@ public DiscordUser(Member member) {
             new LastActivity(),
             new BirthdayData(),
             new MojangProfile(),
-            new GeneratorHistory()
+            new GeneratorHistory(),
+            getDefaultAutoHideGenCommands()
         );
     }
 
+    private static boolean getDefaultAutoHideGenCommands() {
+        // TODO add Logic to retrieve the value from config file
+        // Atleast I think it should probably be a config -socksm
```

Comment:
On second thought I don't think it needs to be retrieved from a config file but rather user data from the database since it's in this class. Also since this class uses the `@Getter` annotation you don't need this method

Reply:
> On second thought I don't think it needs to be retrieved from a config file but rather user data from the database since it's in this class. Also since this class uses the `@Getter` annotation you don't need this method

This is fetching the default for it since I would imagine different people hosting the bot might have different preferences as to what the default is. so plan was later to add it to the config file.



> Leftover TODO

Wasn't aware leaving todo's was bad practice. Was planning on adding it later but first wanted to see what you thought about what I did thus far.

Reply:
If you don't assign a value to a `boolean` variable then it defaults to `false`

Reply:
But yeah my second sentence from above still stands, I don't think it needs to be its own method when the `@Getter` annotation automatically handles it and we're not doing anything special

Reply:
Yeah I realised that part after I replied lol